linux_os/guide/system/permissions/restrictions/sysctl_kernel_core_pattern_empty_string/ansible/shared.yml

```diff
@@ -0,0 +1,32 @@
+# platform = multi_platform_all
```

Comment:
Not sure if the remediation should have Platform similar to prodtype, e.g. rhel9 only.

Reply:
This is fine as per Honza's comment below